src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
```

Comment:
what's the utility of this function? It looks like it doesn't look for the bower.json location...